pkg/aggregator/spandecorator/tag/resource_tag.go

```diff
@@ -54,6 +60,18 @@ func (options *options) Setup(fs *pflag.FlagSet) {
 		"tag a resource (pods, nodes, etc.) span from jsonpath, comma separated. "+
 			"supported setting format: resource[.group]#tagkey1:jsonpath1;tagkey2:jsonpath2, "+
 			"the tag value is parsed from jsonpath. e.g. 'pods#nodes:spec.nodeName;yourTag:status.hostIP,nodes#custom-tag:metadata.name'")
+
+	fs.StringSliceVar(
+		&options.filterVerbs,
+		"resource-tag-filter-verbs",
+		[]string{"create"},
```

Comment:
So if a pod is unhealthy and only has Readiness Probe Failed events, we cannot get its node?

Reply:
it's a option, user can tag all verbs and all span types(object, spec, status, event, audit). by defaut, the object span is tagged, so that we can alse get its node